Classes/BackEnd/AbstractEventMailForm.php

```diff
@@ -201,7 +201,7 @@ private function validateFormData(): bool
      *
      * @return void
      */
-    public function markAsIncomplete()
+    public function markAsIncomplete(): void
```

Comment:
The `void` return type will need to wait until seminars 4.0 as this was introduced in PHP 7.1, but seminars 3 still needs to support PHP 7.0.

https://www.php.net/manual/de/migration71.new-features.php#migration71.new-features.void-functions